herbert_core/herbert3_init.m

```diff
@@ -0,0 +1,144 @@
+function herbert3_init
+% Adds the paths needed by Herbert.
+%
+% Do not call this script directly. Use horace_on or herbert_on 
+% command which enables this script
+% 
+%
+% Is PC and Unix compatible.
+
+% T.G.Perring
```

Comment:
I think this is your authorship, not Toby's, even if you have cut/pasted it from his code

Reply:
woud disagree -- I've fully copied Toby's code into function with new name. If he wants to claim authority -- it is certainly his code.